custom/icds_reports/reports/awc_reports.py

```diff
@@ -1130,6 +1130,7 @@ def base_data(row_data):
         config['data'].append(base_data(row))
 
     config['data'].sort(key=lambda record: record[order], reverse=reversed_order)
+    config['data'] = config['data'][start:(start + length)]
```

Comment:
Not going to block on this so that QA can continue, but it would be great if you could do this pagination at the DB level instead of building the entire dataset in python and then finding the few records you need.